### PR TITLE
updated index.html file and style.css

### PR DESCRIPTION
**Pull request by rmjspecial**
**list of updates made:**

- Add CSS grid to an existing code.
- Update HTML.
- Update CSS.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -58,6 +58,174 @@ <h3>LET’S CONNECT</h3>
...
+      </div>
+    </section>
+
+    <section class="about-myself padding-top-114">
+      <div class="flex about-wrap">
+        <h2 class="section-title">About myself</h2>
+        <p class="headline-profile text-color-primary margin-top-12">I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <a href="#" class="headline-cta margin-top-12">
+          <h3>Let's connect</h3>
+        </a>
```
Comment (ahmetbozaci) [inline index.html:147]: - [ ] You shouldn't wrap the 'let's connect' text with a link tag. Please update it.

File: index.html [hunk 2]
```diff
@@ -58,6 +58,174 @@ <h3>LET’S CONNECT</h3>
...
+              </li>
+              <li class="flex details-list-item">
+                <img class="lang-icon" src="img/icons/html.png" alt="HTML icon">
+                <span class="lang-name">HTML</span>
+              </li>
+              <li class="flex details-list-item">
+                <img class="lang-icon" src="img/icons/CSS.png" alt="CSS icon">
+                <span class="lang-name">CSS</span>
+              </li>
+            </ul>
```
Comment (ahmetbozaci) [inline index.html:202]: - [ ] Please align the languages list with the Languages 

- [ ] I see you gave background color to all bodies. The background color is a little darker than the original design. Please make it a white color to stick to the design.

image

File: index.html [hunk 3]
```diff
@@ -58,6 +58,174 @@ <h3>LET’S CONNECT</h3>
...
+        <h2 class="section-title">About myself</h2>
+        <p class="headline-profile text-color-primary margin-top-12">I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <a href="#" class="headline-cta margin-top-12">
+          <h3>Let's connect</h3>
+        </a>
+        <ul class="social-icons margin-top-12">
+          <li>
+            <a href="#">
+              <img class="social twitter" src="img/icons/twitter.png" alt="Twitter icon">
+            </a>
```
Comment (ahmetbozaci) [inline index.html:152]: ### OPTIONAL
- [ ] Instead of using images, you can use [font awesome ](https://fontawesome.com/)icons.

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# STATUS: Changes Required  ♻️


 @rmjspecial  ✋🏼 

#### Your project is great. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.

___ 

### 💯  Nice to have 💯
- Your git-flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️
___


### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- Your commit messages should always start with capital letters and they should be an imperative tense. I see some of them are not like that. Instead of writing `Added style or Created the file` you should write `Add style or Create the file`
- ***I write it as optional. But this is a [general requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md#:~:text=The%20commit%20message%20should%20be%20written%20in%20the%20imperative%20tense%20and%20capitalized) for every project. So you should pay attention to this in your next projects.***

**OPTIONAL**

- You can change them by using `git-rebase`. If you want to know how to change ı leave a few resources.
- [Video - How to change commit messages](https://www.loom.com/share/00771f369b7b47fc968c019ecf0d3ba6)
- [Article - How to create good commit messages](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8/)
___

### Cheers and Happy coding! 💻 👏
almost


___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Review by uwadonat — APPROVED** [review]
Hi @[rmjspecial](https://github.com/rmjspecial), 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
appr

# Status: Approved ✔️✔️✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**